CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ More detailed release notes can be found on the [releases page](https://github.c
 
 * [fyne-cli] Add support for passing custom build tags (#1538)
 
+### Changed
+
+* Run validation on content change instead of on each Refresh in widget.Entry
+* Run onValidationChanged() when calling entry.SetValidationError()
```

Comment:
The way this is expressed it's an internal detail. Either it does not belong here, or it should describe why it is a developer facing change. Maybe it's part of the line above?
Not sure that having it as a change as well as a fix is required? We don't list changes that are bug fixes.

Reply:
Oh, sorry. Was pretty sure that I had fixed this.